### PR TITLE
Nov-2021-correction

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2021.acl.xml [hunk 1]
```diff
@@ -5182,9 +5182,11 @@
...
-      <url hash="18c525c0">2021.acl-long.344</url>
+      <url hash="161854c0">2021.acl-long.344</url>
       <doi>10.18653/v1/2021.acl-long.344</doi>
       <bibkey>tian-etal-2021-dependency</bibkey>
+      <revision id="1" href="2021.acl-long.344v1" hash="18c525c0"/>
+      <revision id="2" href="2021.acl-long.344v2" hash="161854c0" date="2021-11-09">updated equation 8 and figure 2</revision>
```
Comment (mjpost) [inline data/xml/2021.acl.xml:5189]: Please capitalize this properly: "Updated equation 8 and figure 2."

File: data/xml/2021.blackboxnlp.xml [hunk 2]
```diff
@@ -278,11 +278,13 @@
...
-      <url hash="08714d57">2021.blackboxnlp-1.22</url>
+      <url hash="fffb1242">2021.blackboxnlp-1.22</url>
       <bibkey>talukdar-etal-2021-training</bibkey>
       <pwcdataset url="https://paperswithcode.com/dataset/glue">GLUE</pwcdataset>
       <pwcdataset url="https://paperswithcode.com/dataset/sst">SST</pwcdataset>
       <pwcdataset url="https://paperswithcode.com/dataset/cola">CoLA</pwcdataset>
+      <revision id="1" href="2021.blackboxnlp-1.22v1" hash="08714d57"/>
+      <revision id="2" href="2021.blackboxnlp-1.22v2" hash="fffb1242" date="2021-11-15">Section 3.2 and author email address fix</revision>
```
Comment (mjpost) [inline data/xml/2021.blackboxnlp.xml:287]: "Updated section 3.2 and fixed an author's email address."

File: data/xml/2021.emnlp.xml [hunk 3]
```diff
@@ -4308,9 +4308,11 @@
...
-      <url hash="981bdc4c">2021.emnlp-main.318</url>
+      <url hash="1b80efdd">2021.emnlp-main.318</url>
       <bibkey>jing-etal-2021-seeking</bibkey>
       <pwcdataset url="https://paperswithcode.com/dataset/aste-data-v2">ASTE-Data-V2</pwcdataset>
+      <revision id="1" href="2021.emnlp-main.318v1" hash="981bdc4c"/>
+      <revision id="2" href="2021.emnlp-main.318v2" hash="1b80efdd" date="2021-11-18">Deleted fundings sources in acknolowdgement</revision>
```
Comment (mjpost) [inline data/xml/2021.emnlp.xml:4315]: Typos: fundings → funding, acknowledgment, end with period.

File: data/xml/2021.emnlp.xml [hunk 4]
```diff
@@ -7482,9 +7484,11 @@
...
-      <url hash="ecf6ae2a">2021.emnlp-main.557</url>
+      <url hash="a885570e">2021.emnlp-main.557</url>
       <attachment type="Software" hash="e6eebc73">2021.emnlp-main.557.Software.zip</attachment>
       <bibkey>thawani-etal-2021-numeracy</bibkey>
+      <revision id="1" href="2021.emnlp-main.557v1" hash="ecf6ae2a"/>
+      <revision id="2" href="2021.emnlp-main.557v2" hash="a885570e" date="2021-11-09">added acknowledgements</revision>
```
Comment (mjpost) [inline data/xml/2021.emnlp.xml:7491]: "Added acknowledgments."

File: data/xml/2021.emnlp.xml [hunk 5]
```diff
@@ -11175,8 +11179,10 @@
...
       <abstract>We report two essential improvements in readability assessment: 1. three novel features in advanced semantics and 2. the timely evidence that traditional ML models (e.g. Random Forest, using handcrafted features) can combine with transformers (e.g. RoBERTa) to augment model performance. First, we explore suitable transformers and traditional ML models. Then, we extract 255 handcrafted linguistic features using self-developed extraction software. Finally, we assemble those to create several hybrid models, achieving state-of-the-art (SOTA) accuracy on popular datasets in readability assessment. The use of handcrafted features help model performance on smaller datasets. Notably, our RoBERTA-RF-T1 hybrid achieves the near-perfect classification accuracy of 99%, a 20.3% increase from the previous SOTA.</abstract>
-      <url hash="68ff98b2">2021.emnlp-main.834</url>
+      <url hash="75ab197c">2021.emnlp-main.834</url>
       <bibkey>lee-etal-2021-pushing</bibkey>
+      <revision id="1" href="2021.emnlp-main.834v1" hash="68ff98b2"/>
+      <revision id="2" href="2021.emnlp-main.834v2" hash="75ab197c" date="2021-11-09">typo fix</revision>
```
Comment (mjpost) [inline data/xml/2021.emnlp.xml:11185]: "Corrected a typo."

File: data/xml/2021.findings.xml [hunk 6]
```diff
@@ -9326,8 +9326,10 @@
...
-      <url hash="8b37358f">2021.findings-emnlp.253</url>
+      <url hash="3309b723">2021.findings-emnlp.253</url>
       <bibkey>matero-etal-2021-melt-message</bibkey>
+      <revision id="1" href="2021.findings-emnlp.253v1" hash="8b37358f"/>
+      <revision id="2" href="2021.findings-emnlp.253v2" hash="3309b723" date="2021-11-09">fixed typo, citation and table</revision>
```
Comment (mjpost) [inline data/xml/2021.findings.xml:9332]: "Corrected a typo, a citation, and a table."

File: data/xml/2021.findings.xml [hunk 7]
```diff
@@ -10259,9 +10261,11 @@
...
-      <url hash="55ad08da">2021.findings-emnlp.328</url>
+      <url hash="4b315080">2021.findings-emnlp.328</url>
       <bibkey>iso-etal-2021-convex-aggregation</bibkey>
       <pwccode url="https://github.com/megagonlabs/coop" additional="false">megagonlabs/coop</pwccode>
+      <revision id="1" href="2021.findings-emnlp.328v1" hash="55ad08da"/>
+      <revision id="2" href="2021.findings-emnlp.328v2" hash="4b315080" date="2021-11-16">Fix falsely classified existing approach and its relevant content</revision>
```
Comment (mjpost) [inline data/xml/2021.findings.xml:10268]: End with period.

File: data/xml/2021.law.xml [hunk 8]
```diff
@@ -54,8 +54,10 @@
       <author><first>Meagan</first><last>Vigus</last></author>
       <pages>36–45</pages>
       <abstract>We present AutoAspect, a novel, rule-based annotation tool for labeling tense and aspect. The pilot version annotates English data. The aspect labels are designed specifically for Uniform Meaning Representations (UMR), an annotation schema that aims to encode crosslingual semantic information. The annotation tool combines syntactic and semantic cues to assign aspects on a sentence-by-sentence basis, following a sequence of rules that each output a UMR aspect. Identified events proceed through the sequence until they are assigned an aspect. We achieve a recall of 76.17% for identifying UMR events and an accuracy of 62.57% on all identified events, with high precision values for 2 of the aspect labels.</abstract>
-      <url hash="73da1acb">2021.law-1.4</url>
+      <url hash="058b0f56">2021.law-1.4</url>
       <bibkey>chen-etal-2021-autoaspect</bibkey>
+      <revision id="1" href="2021.law-1.4v1" hash="73da1acb"/>
+      <revision id="2" href="2021.law-1.4v2" hash="058b0f56" date="2021-11-09">added a sponsor in acknowledgements</revision>
```
Comment (mjpost) [inline data/xml/2021.law.xml:60]: "Added a sponsor in the acknowledgments."

File: data/xml/2021.mrl.xml [hunk 9]
```diff
@@ -275,8 +275,11 @@
       <author><first>Lucie</first><last>Flek</last></author>
       <pages>242–244</pages>
       <abstract>Interpolation-based regularisation methods have proven to be effective for various tasks and modalities. Mixup is a data augmentation method that generates virtual training samples from convex combinations of individual inputs and labels. We extend Mixup and propose DMix, distance-constrained interpolative Mixup for sentence classification leveraging the hyperbolic space. DMix achieves state-of-the-art results on sentence classification over existing data augmentation methods across datasets in four languages.</abstract>
-      <url hash="47db7ca9">2021.mrl-1.21</url>
+      <url hash="8996a24d">2021.mrl-1.21</url>
       <bibkey>sawhney-etal-2021-dmix</bibkey>
+      <revision id="1" href="2021.mrl-1.21v1" hash="47db7ca9"/>
+      <revision id="2" href="2021.mrl-1.21v2" hash="8996a24d" date="2021-11-16">Retracted.</revision>
+      <retracted date="2021-11-16">author wants to withdraw paper</retracted>
```
Comment (mjpost) [inline data/xml/2021.mrl.xml:282]: This explanation should be more informative. What is the reason for the retraction?

File: data/xml/2021.mrl.xml [hunk 10]
```diff
@@ -286,8 +289,11 @@
       <author><first>Debdoot</first><last>Mukherjee</last></author>
       <pages>245–247</pages>
       <abstract>There have been efforts in cross-lingual transfer learning for various tasks. We present an approach utilizing an interpolative data augmentation method, Mixup, to improve the generalizability of models for part-of-speech tagging trained on a source language, improving its performance on unseen target languages. Through experiments on ten languages with diverse structures and language roots, we put forward its applicability for downstream zero-shot cross-lingual tasks.</abstract>
-      <url hash="f6ab5075">2021.mrl-1.22</url>
+      <url hash="2db89a52">2021.mrl-1.22</url>
       <bibkey>thakkar-etal-2021-sequence</bibkey>
+      <revision id="1" href="2021.mrl-1.22v1" hash="f6ab5075"/>
+      <revision id="2" href="2021.mrl-1.22v2" hash="2db89a52" date="2021-11-16">Retracted.</revision>
+      <retracted date="2021-11-16">author wants to withdraw paper</retracted>
```
Comment (mjpost) [inline data/xml/2021.mrl.xml:296]: Please elaborate.

File: data/xml/W19.xml [hunk 11]
```diff
@@ -15877,8 +15877,10 @@ One of the references was wrong therefore it is corrected to cite the appropriat
       <author><first>Francis</first><last>Tyers</last></author>
       <author><first>Jonathan</first><last>Washington</last></author>
       <pages>24–31</pages>
-      <url hash="1fe0051c">W19-6805</url>
+      <url hash="9892f661">W19-6805</url>
       <bibkey>gokirmak-etal-2019-machine</bibkey>
+      <revision id="1" href="W19-6805v1" hash="1fe0051c"/>
+      <revision id="2" href="W19-6805v2" hash="9892f661" date="2021-11-09">responded to reviewer comments</revision>
```
Comment (mjpost) [inline data/xml/W19.xml:15883]: Please punctuate. Also, did the authors elaborate? This is a bit too vague.

Comment (xinru1414) [reply data/xml/W19.xml:15883]: Done. I think they submitted the wrong version the first time. The final version addressed reviewer's comments and is intended for publication.

**Review by danielgildea — APPROVED** [review]

**Pull request merged**